blog/docs/articles/using-sideloaded-images-with-kind.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Using Sideloaded Images with Kind
+linkTitle: Using-Sideloaded -Images-with-Kind
+author: "Ed Jung"
+date: 2022-05-26
+description: How to use sideloaded images with Knative on Kind
+type: "blog"
+---
```

Comment:
```suggestion
# Using Sideloaded Images with Kind

**Author: Ed Jung**
**Date: 2022-05-26**
```

Old frontmatter format no longer required, we're in the process of updating the templates, but please use this suggestion for now.